README.md

```diff
@@ -123,8 +158,68 @@ It includes:
 More details about vetters can be found in the individual vetters package
 documentation.
 
+<br>
+<br>
+
 ## Contributing
 Individuals or business entities who contribute to this project must have
 completed and submitted the [F5® Contributor License Agreement](https://github.com/aspenmesh/cla/raw/master/f5-cla.pdf)
 to [cla@aspenmesh.io](mailto:cla@aspenmesh.io) prior to their code submission
 being included in this project. Please include your github username in the CLA email.
+
+### Build Prerequisites
+To build Istio-Vet locally, you will need to install the following:
+
+* A [Go environment](https://golang.org/doc/install).
+* Install [dep](https://golang.github.io/dep/docs/installation.html)
+* Install [Protobuf](https://github.com/golang/protobuf)
```

Comment:
Let's break this up in two sections for Mac and Linux users so that they can easily follow along.

Reply:
Instructions for protoc: http://google.github.io/proto-lens/installing-protoc.html